.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-  - 5.4
```

Comment:
The current problem is that CakePHP 3.2 made backwards incompatible changes to the `EntityInterface`. This means that this plugin has to be either CakePHP 3.2 up or specifically require a version lower than 3.2.

Those backwards incompatible changes can be seen at https://github.com/cakephp/cakephp/commit/ad386871f27abcdc35e8d2b170637d01d4d790c2#diff-fcf8477a981a8f104e4d2d3d2b4ffdbfL65 and can be noticed in https://travis-ci.org/UseMuffin/Webservice/jobs/106965645#L319